modules/axtask/src/run_queue.rs

```diff
@@ -1,45 +1,202 @@
 use alloc::collections::VecDeque;
 use alloc::sync::Arc;
-use kspin::SpinNoIrq;
+use core::mem::MaybeUninit;
+#[cfg(feature = "smp")]
+use core::sync::atomic::{AtomicUsize, Ordering};
```

Comment:
These types only used in `select_run_queue_index`, we can import them within the function.